src/ports/postgres/modules/lda/test/lda.sql_in

```diff
@@ -438,7 +444,9 @@ select assert(array_upper(perplexity_iters,1) <= 5, 'Perplexity iterations are d
 select assert(perplexity[1] > 0 , 'Perplexity value should be greate than 0') from lda_model ;
 
 
--- Test to check if the perplexity_toll is greater than the difference between two perplexity iterations --
+-- Test: If the difference between the last two iterations is less than the perplexity_tol, the iterations training will stop --
```

Comment:
Instead of saying `last two iterations` we can just say `If the perplexity difference between any two iterations is less than the perplexity_tol, we will stop training.`